package.json

```diff
@@ -177,15 +177,14 @@
     "nodemon": "^3.1.10",
     "prettier": "^3.6.2",
     "prettier-plugin-jinja-template": "^2.1.0",
-    "sass": "^1.89.2",
+    "sass": "^1.93.3",
     "sinon": "^19.0.5",
     "supertest": "^7.1.4",
-    "ts-jest": "^29.4.0",
-    "typescript": "^5.8.3",
+    "ts-jest": "^29.4.5",
+    "typescript": "^5.9.3",
     "wiremock": "^3.13.1"
   },
   "overrides": {
-    "undici": "^6.21.3",
-    "@babel/helpers": "^7.26.10"
+    "validator": "^13.15.20"
```

Comment:
Override for class-validator (which doesn't seem to be getting maintained much anymore).